it/it-tests/src/test/java/it/authorisation/AuthenticationTest.java

```diff
@@ -89,9 +91,9 @@ public void basic_authentication_based_on_login_and_password() {
     createUser(login, name, password);
 
     // authenticate
-    WsClient wsClient = new WsClient(newHttpConnector().url(ORCHESTRATOR.getServer().getUrl()).login(login).password(password).build());
-    String response = wsClient.execute(newGetRequest("api/authentication/validate"));
```

Comment:
I prefer to hide the WsConnector when using the WsClient. It's used for initialization, but after that I'd much prefer a wsClient.call(WsRequest).
The same, I prefer static constructor method than new GetRequest and new PostRequest objects.